reqs/admin.py

```diff
@@ -1,7 +1,31 @@
+from dal_select2_taggit.widgets import TaggitSelect2
+from django import forms
 from django.contrib import admin
 
 from reqs.models import Keyword, Policy, Requirement
 
 admin.site.register(Policy)
-admin.site.register(Requirement)
 admin.site.register(Keyword)
+
+
+class TaggitWidget(TaggitSelect2):
+    """Account for commas in tags by wrapping each entry in double quotes"""
+    def value_from_datadict(self, data, files, name):
+        values = data.getlist(name)
+        values = [v.replace('"', '') for v in values]   # remove double quotes
+        values = ['"{0}"'.format(v) for v in values]
+        return ','.join(values)
+
```

Comment:
I can imagine a tag where the quotation marks are important; should we do something like this?
```
def handle_quotation_marks(value):
    """Account for commas and quotation marks in tags."""
    num_marks = value.count('"')
    if num_marks % 2 != 0:
        value = value.replace('"', '')
    else:
        while '"' in value:
            marks = ("“", "”")
            value = value.replace('"', marks[value.count('"') % 2], 1)

    return '"{0}"'.format(value)
```
(I have this + tests locally and can add it if you think this is a reasonable idea.)